crates/glaredb/src/rpc_proxy.rs

```diff
@@ -16,6 +17,17 @@ pub struct RpcProxy {
     handler: RpcProxyHandler<CloudAuthenticator>,
 }
 
+#[derive(Clone, Debug, ValueEnum, Default)]
+#[clap(rename_all = "lower")]
+pub enum TLSMode {
+    Required,
+    Optional,
+
+    /// Note: in the future, this will be 'required' by default
```

Comment:
```suggestion
    /// Note: in the future, this will be 'Required' by default
```

just to match value above, maybe? Or not...doesn't matter